MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
-	 - This is important for both understandability, readability and accessibility of MIPs.
-3.  **Avoid overlap:** Multiple MIPs should not implement the same type of behaviour independently. For instance, there should not be two separate but interchangeable ways to do collateral onboarding.
-	- This way the primary and best-understood process for each particular behaviour will be fairly available to everyone, without risking having a knowledge gap that makes it possible for some actors with better access to information to use different and potentially better processes
-4. **Clarity:** A MIP must not have equally valid conflicting interpretations. MIP Authors and MIP Editors must strive to reduce ambiguity. A MIP must be as clear and easy to understand as possible.
-	- Any ambiguous MIPs are likely to cause contention or confusion in the future. Making everything as clear as possible also aids readability and helps to mitigate the risk of hidden attacks.
-5. **Brevity:** A MIP must be as short as possible, including only that which is essential given the other core principles.
-	- The shorter MIPs are the more likely participants in governance are to read them in full. This also serves to reduce the surface area for hidden attacks.
-	
----
+   If the verification is successful, the MIP Editor:
+
+   - Approves the MIP and assigns it a formal MIP number.
+   - Merges in the PR.
 
-### MIP0c3: The MIP Lifecycle
+3. **Request for Comments (RFC)**: A period of reviewing by the community and attendant redrafting begins. The minimum duration of this period is determined by two variables:
 
-**The MIP Lifecycle Flow and MIP Statuses**
+   - `Feedback Period`: 3 months
+   - `Frozen Period`: 1 month
 
-![mip_life_cycle](https://user-images.githubusercontent.com/32653033/79086728-19d93900-7d0b-11ea-8086-c255d919096c.png)
+   These periods can overlap. See `MIP0c1` for their definitions
 
+4. **Fulfilled Feedback Period Requirements:** After the MIP has fulfilled the RFC phase, it is ready for Formal Submission.
 
-**MIP Status Criteria**  
+5. **Formal Submission (FS):** At this point the MIP Author submits their MIP to the Governance Cycle by posting it to the [*Formal Submission* forum category](https://forum.makerdao.com/c/mips/fs/16) within the [formal submission window of a Governance Cycle](https://github.com/makerdao/mips/blob/master/MIP16/mip16.md).
 
-**1. Conception:** The lifecycle of a MIP begins when the MIP proposal is posted on the Maker forum. However, in order for a MIP to move to the next stage, it needs to satisfy the transition criteria (1) described below:
+   > A MIP can be re-submitted to the formal submission process a maximum of 2 additional times (3 total), without having to go through phase 1- 4 again, if it failed to pass due to legitimate external reasons (e.g. got bundled in a governance poll or executive vote with a controversial proposal - subject to the governance facilitators judgement).
```

Comment:
This feels like it should be moved further down? Maybe outside of the 'ideal flow'

Reply:
Yep.

Maybe under #### Resubmission